subxt/src/backend/unstable/mod.rs

```diff
@@ -444,9 +451,7 @@ impl<T: Config + Send + Sync + 'static> Backend<T> for UnstableBackend<T> {
         &self,
     ) -> Result<StreamOfResults<(T::Header, BlockRef<T::Hash>)>, Error> {
         self.stream_headers(|ev| match ev {
-            FollowEvent::Initialized(ev) => {
-                vec![ev.finalized_block_hash]
-            }
+            FollowEvent::Initialized(init) => init.finalized_block_hashes,
```

Comment:
When we add reconnection stuff in, I guess we might tweak eg this stream_finalized function to try and pick up from where they left off (ie keep track of the latest finalized block hash and if see an init event, ignore any blocks we get back before this latest one).

But for now it's all good!